### PR TITLE
Changes to relieve bugs found during testing

### PR DESCRIPTION
**Pull request by khardsonhurley**
- Removed Entities, since this endpoint no longer exists.
- Added new classes to provide easily readible objects.
- Took requester out of __init__ in events module.

### CODE REVIEW

File: pygerduty/events.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ def __str__(self):
 
 
 class Events(object):
-    def __init__(self, service_key, requester):
+    def __init__(self, service_key):
```
Comment (gmjosack) [inline pygerduty/events.py]: I think it's still valuable to allow a Requester to be passed. Maybe something like 

```python
    def __init__(self, service_key, requester=None):
        self.service_key = service_key
        if requester is None:
            self.requester = pygerduty.common.Requester()
        else:
            self.requester = requester
```

Comment (khardsonhurley) [inline pygerduty/events.py]: ok will make this change!

File: pygerduty/events.py [hunk 2]
```diff
@@ -24,7 +25,10 @@ def __str__(self):
 class Events(object):
     def __init__(self, service_key, requester):
```
Comment (gmjosack) [inline pygerduty/events.py]: Can you set a default value of None here?

**Comment by gmjosack** [comment]
Is the `Entries` class still used or can it be removed now?

**Comment by khardsonhurley** [comment]
From what I can tell only LogEntries class will be used now (https://v2.developer.pagerduty.com/v2/page/api-reference#!/Log_Entries/get_log_entries). I will go ahead and remove.

**Comment by khardsonhurley** [comment]
Actually, was entries meant to encapsulate the `rendered_schedule_entries` when you wrote the library? This has not since been removed: 

https://v2.developer.pagerduty.com/v2/page/api-reference#!/Schedules/get_schedules_id
https://v1.developer.pagerduty.com/documentation/rest/schedules/show
screenshot 2017-07-10 16 46 22

**Comment by gmjosack** [comment]
Pretty sure it was specifically for entries returned by the api that was removed.

**Pull request merged**